db/app.js

```diff
@@ -4,18 +4,30 @@ const {
 	getAllEndpoints,
   getAllArticles,
 	getArticleById,
-  getCommentsById
+  getCommentsById,
+  postComments
 } = require("./controller");
+
 const app = express();
+app.use(express.json());
 
 app.get("/api/topics", getAllTopics);
 app.get("/api", getAllEndpoints);
 app.get("/api/articles/:article_id", getArticleById);
 app.get("/api/articles/:article_id/comments", getCommentsById )
 app.get("/api/articles", getAllArticles )
+app.post("/api/articles/:article_id/comments", postComments)
 
 // Error Handling
```

Comment:
at this point, it might be nice to separate your error handlers to a separate controller? Just so this app.js file doesn't get bloated :)